docker/.env

```diff
@@ -42,3 +42,7 @@ REDIS_PORT=6379
 FLASK_ENV=development
 SUPERSET_ENV=development
 SUPERSET_LOAD_EXAMPLES=yes
+
+SUPERSET_CONFIG=tests.superset_test_config
+SUPERSET_TESTENV=true
+ENABLE_REACT_CRUD_VIEWS=false
```

Comment:
I set CRUD_VIEWS to false because of failing tests:
```self = <tests.core_tests.TestCore testMethod=test_tablemodelview_list>
    def test_tablemodelview_list(self):
        self.login(username="admin")
        url = "/tablemodelview/list/"
        resp = self.get_resp(url)
        # assert that a table is listed
        table = db.session.query(SqlaTable).first()
>       assert table.name in resp
E       AssertionError: assert 'energy_usage' in '\n\n\n<!DOCTYPE html>\n\n\n\n\n<html>\n  <head>\n    <title>\n      \n        \n          Superset\n        \n      \...\n\n    \n  \n  \n  <!-- Bundle js crudViews START -->\n  \n  <!-- Bundle js crudViews END -->\n\n\n  </body>\n</html>'
E        +  where 'energy_usage' = energy_usage.name```